src/utils/ddb/ddb_parse.h

```diff
@@ -54,6 +54,6 @@ int ddb_parse_dtx_id(const char *dtx_id_str, struct dtx_id *dtx_id);
 
 /* Parse a string representing a date into an epoch */
 int
-ddb_date2epoch(const char *date, uint64_t *epoch);
+ddb_date2cmt_time(const char *date, uint64_t *cmt_time);
```

Comment:
Please refresh the comment to match the changes.

Reply:
> Please refresh the comment to match the changes.

- [ ] Fix comment